addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1889,8 +1889,10 @@ const ListUserValueWidget = UserValueWidget.extend({
      * @private
      * @param {string || integer} id
      * @param {string} text
+     * @param {Object} recordData key, values that will be added
```

Comment:
Optional params should be documented with the variable between `[]`. It's a mistake that `text` does not use that.